contracts/eosio.bios/include/eosio.bios/eosio.bios.hpp

```diff
@@ -1,19 +1,24 @@
+/**
+ *  @copyright defined in eosio.cdt/LICENSE.txt
+ */
+
 #pragma once
-#include <eosiolib/action.hpp>
-#include <eosiolib/crypto.h>
-#include <eosiolib/eosio.hpp>
-#include <eosiolib/privileged.hpp>
-#include <eosiolib/producer_schedule.hpp>
-#include <eosiolib/fixed_bytes.hpp>
+
+#include <eosio/action.hpp>
+#include <eosio/crypto.hpp>
+#include <eosio/eosio.hpp>
+#include <eosio/fixed_bytes.hpp>
+#include <eosio/privileged.hpp>
+#include <eosio/producer_schedule.hpp>
 
 namespace eosio {
```

Comment:
Change this back to `::capi_checksum256` and place a comment above this section stating "needed until these make it into eosio.cdt" and add the include `#include <eosio/../../capi/eosio/crypto.h>`.